### ISSUE TITLE
Release TwistedLore-0.3.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2363]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#2363|
|Type|enhancement|
|Created|2007-01-06 22:20:51Z|

<details><summary>Searchable metadata</summary>

```
trac-id__2363 2363
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1168122051000000 1168122051000000
changetime__1168134419000000 1168134419000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix removed owner|
|-|-|

Ready for review at releases/twisted-lore-0.3.x-2363

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @radix|
|-|-|

Looks good

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix set status to `closed`|
|-|-|

(In [19224]) Merge releases/twisted-lore-0.3.x-2363

Author: release automation, radix
Reviewer: exarkun
Fixes [#2363](https://github.com/twisted/twisted/issues/2363)

Version numbers and NEWS from the release of lore 0.3.0

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
